src/main/java/com/appbuddy/buddypasswordmanager/utils/BPMUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.appbuddy.buddypasswordmanager.utils;
+
+import com.appbuddy.buddypasswordmanager.gui.EntryListTable;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+/**
+ *
+ */
+public class BPMUtils {
+    public static void populateTable() {
+        int row = 0;
+        int col = 0;
+        String nameOfUser = System.getProperty("user.name");
+        System.out.println("The name of the user is " + nameOfUser);
+        String entry;
+
+        File filePath = new File("/Users/" + nameOfUser + "/Documents/Buddy Information.txt");
```

Comment:
string interpolation would be nice here.

Reply:
It would be... maybe next App Buddy day...lol